MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -283,13 +281,14 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel(vtkMRMLMarkupsT
   }
   int numberOfMarkups = markups->GetNumberOfFiducials();
 
-  if(numberOfMarkups < 1)
+  if( numberOfMarkups < 1 )
   {
-    if(markupsToModelModuleNode->GetModelNode()!=NULL)
+    if ( markupsToModelModuleNode->GetModelNode() != NULL && markupsToModelModuleNode->GetModelNode()->GetPolyData() != NULL )
     {
+      // TODO: Review this code, see if it is really needed...
```

Comment:
Please check if VTK warnings are logged before you add any markups or after you remove all markups and you change display properties of the output model. Since Slicer now uses VTK7, there could have been some changes that made the warnings disappear, so maybe creating an 'almost-empty polydata' is not needed anymore.